.circleci/config.yml

```diff
@@ -61,6 +69,10 @@ workflows:
   build:
     jobs:
       - dependencies
+      # Run linting
+      - lint:
+          requires:
```

Comment:
Shouldn't this go into the job definition for lint above?

Reply:
I don't think so. Requires is part of the workflow:
https://circleci.com/docs/2.0/configuration-reference/#requires